src/options.js

```diff
@@ -4,6 +4,7 @@ const validate = require("jest-validate").validate;
 const deprecatedConfig = require("./deprecated");
 
 const defaults = {
+  cursorOffset: -1,
```

Comment:
Yeah, `undefined` would make more sense. However, when I tried it, I got a validation error:

```
 FAIL  tests/cursor/jsfmt.spec.js
  ● translates cursor correctly in basic case

    Error
      ● Validation Error:

        Option "cursorOffset" must be of type:
          undefined
        but instead received:
          number

        Example:
        {
          "cursorOffset": undefined
        }


  ✕ translates cursor correctly in basic case (3ms)
```

Here's the diff I tested:

<details>

```diff
diff --git a/index.js b/index.js
index 5a66c9cb..e9e67231 100644
--- a/index.js
+++ b/index.js
@@ -64,7 +64,7 @@ function formatWithCursor(text, opts, addAlignmentSize) {
   }
 
   let cursorOffset;
-  if (opts.cursorOffset >= 0) {
+  if (opts.cursorOffset !== undefined) {
     const cursorNodeAndParents = findNodeAtOffset(ast, opts.cursorOffset);
     const cursorNode = cursorNodeAndParents.node;
     if (cursorNode) {
diff --git a/src/options.js b/src/options.js
index 73d40426..956168af 100644
--- a/src/options.js
+++ b/src/options.js
@@ -4,7 +4,7 @@ const validate = require("jest-validate").validate;
 const deprecatedConfig = require("./deprecated");
 
 const defaults = {
-  cursorOffset: -1,
+  cursorOffset: undefined,
   rangeStart: 0,
   rangeEnd: Infinity,
   useTabs: false,
```
</details>

Is there a good way to get around this?